feature-group-definitions/fetch.dist.yml

```diff
@@ -0,0 +1,77 @@
+# Generated from: fetch.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Fetch (initial support)
+description: Fetch makes HTTP requests.
+spec: https://fetch.spec.whatwg.org/
+caniuse: fetch
+status:
+  baseline: high
+  baseline_low_date: 2018-09-04
+  baseline_high_date: 2021-03-04
+  support:
+    chrome: "69"
```

Comment:
I think these versions should match https://developer.mozilla.org/en-US/docs/Web/API/fetch#browser_compatibility. Can you remove parts of the feature until it does match?

Reply:
OK, for posterity, this is what we're losing to match the top row of the MDN page or caniuse:

- `api.Headers.entries`
- `api.Headers.forEach`
- `api.Headers.keys`
- `api.Headers.values`
- `api.Headers.@@iterator`
- `api.Request.Request.cross_origin_stripped`
- `api.Request.Request.init_referrer_parameter`
- `api.Request.cache`
- `api.Request.destination`
- `api.Request.integrity`
- `api.Request.redirect`
- `api.Request.referrer`
- `api.Request.referrerPolicy`
- `api.Response.error_static`
- `api.Response.redirect_static`
- `api.Response.redirected`
- `api.fetch.init_signal_parameter`

If you squint, you can see some distinct features here:

- iterable `Headers` (this one feels like a sharp edge though)
- redirects/redirect mode
- referrers (weirdly, that `cross_origin_stripped` is part of this)
- abortable fetch

Unfortunately, there's a bit of a grab bag with `Request.cache`, `destination`, and `integrity` though they all look a bit obscure to me.

Reply:
Yeah, I think that we'll have a hard time in many cases calling the things that ship after the initial support features. It's a trickle of small things that make the whole thing better, but often not "features".